### PR TITLE
Add Appveyor CI

### PR DESCRIPTION
**Pull request by nitriques**
We now also try to build and test on windows.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Test against the latest version of this Node.js version
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "8"
+    - nodejs_version: "9"
```
Comment (fhamon) [inline .appveyor.yml]: Do we really want it to run on an non-lts version @nitriques ?

Comment (nitriques) [reply .appveyor.yml]: Yes. When 10 goes out, we will switch to 10 and add 11 when possible.
We should always test against the latest version. I am open to let it fail.

Comment (fhamon) [reply .appveyor.yml]: Perfect. If the tests fails with a non-lts version, will the badge turn in an error state?

Can (and should?) we ignore non-lts build error to keep the badge green?

Comment (nitriques) [reply .appveyor.yml]: It is safe to ignore IHMO

Check https://www.appveyor.com/docs/build-configuration/#failing-strategy (matrix.allow_failure)

And  https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail  (matrix.allow_failure)

**Pull request merged**